requirements_all.txt

```diff
@@ -1682,7 +1682,7 @@ tikteck==0.4
 todoist-python==7.0.17
 
 # homeassistant.components.toon
-toonapilib==3.0.4
+toonapilib==3.0.7
```

Comment:
Please use 3.0.9 as 3.0.7 is broken for 3.5 compatibility.

Reply:
Resolved